daq/forchestrator.py

```diff
@@ -95,6 +99,7 @@ def get_topology(self, path, params):
     HTTP.map_request('topology', FORCH.get_topology)
     HTTP.map_request('switches', FORCH.get_switches)
     HTTP.map_request('switch', FORCH.get_switch)
+    HTTP.map_request('active-host-route', FORCH.get_active_host_route)
```

Comment:
simplify request to 'host_route'

generally avoid '-' in any kind of key since it will confuse some things greatly.

Reply:
Done